serverless.yml

```diff
@@ -7,13 +7,27 @@ provider:
   runtime: nodejs14.x
 
 functions:
+  addProduct:
+    handler: lambda-functions/products/addProduct.handler
+    environment:
+      PRODUCT_TABLE: ${self:custom.ProductTable.name}
+
+    events:
+      - http:
+          path: "ecommerce-vs/products"
+          method: post
+          cors: true
+
   getProductsList:
     handler: lambda-functions/products/getProductsList.handler
+    environment:
+      PRODUCT_TABLE: ${self:custom.ProductTable.name}
```

Comment:
@parabolabam it should be 2 tables- products and stocks , your stocks table is missing in implementation

Reply:
According to AWS NoSQL Design Best Practices it's recommended to keep related data close 
```
Keep related data together.   Research on routing-table optimization 20 years ago found that "locality of reference" was the single most important factor in speeding up response time: keeping related data together in one place. This is equally true in NoSQL systems today, where keeping related data in close proximity has a major impact on cost and performance. Instead of distributing related data items across multiple tables, you should keep related items in your NoSQL system as close together as possible.
```

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-general-nosql-design.html

Also, it's not scalable, but what in fact scanning too tables for related will take much more time and consume much more resources than keeping data close together in one table knowing they are tightly related.